content/applications/inventory_and_mrp/inventory/product_management/inventory_valuation/valuation_by_lots.rst

```diff
@@ -0,0 +1,179 @@
+create the :guilabel:`Lot/Serial Number`, set the :guilabel:`Counted Quantity`, and click
+:guilabel:`Apply`.
+
+To view the valuation layer, go to :menuselection:`Inventory app --> Reporting --> Valuation`. The
+:guilabel:`Total Value` per unit will match the cost on the product form.
+
+.. example::
+   Continuing the example in the table above, when the product cost is `$15`, the valuation for a
+   newly created `LOT3` will also be `$15`.
+
+   .. image:: valuation_by_lots/create-new.png
+      :align: center
+      :alt: Show inventory adjustment valuation.
+
+.. _inventory/product_management/valuation-cost-existing:
+
+Existing lot/serial
+-------------------
+
+When adjusting the quantity of an existing lot/serial number, the value is based on the most recent
+valuation layer that specific lot/serial number.
+
+.. example::
+   Continuing the example in the table above, when LOT 1's quantity is updated from `1.00` to
+   `2.00`, the additional quantity is valued at `$10`, reflecting the latest valuation layer for
+   `LOT 1`.
+
+   .. figure:: valuation_by_lots/existing.png
+      :align: center
+      :alt: Show valuation of LOT 1 getting updated.
+
+      The inventory adjustment (top line) is valued the same as LOT 1 (bottom line).
+
+View valuation
+==============
+
+To find the average cost of a specific lot/serial number, go to :menuselection:`Inventory app -->
+Products --> Lots/Serial Numbers` and select the desired record. The :guilabel:`Cost` field displays
+the average cost of the lot/serial number.
+
+.. image:: valuation_by_lots/lot.png
+   :align: center
+   :alt: Show cost of the lot/serial number.
+
+Valuations of a lot/serial number can also be viewed through the valuation report or through the
+lot/serial number form page.
+
+Valuation report
+----------------
+
+Display the valuation of lots and serial numbers in the database by going to
+:menuselection:`Inventory app --> Reporting --> Valuation`.
+
+On the :guilabel:`Stock Valuation` report, click the search bar, and in the :guilabel:`Group By`
+section of the resulting drop-down menu, select :guilabel:`Lot/Serial number`.
+
+.. tip::
+   Click the :icon:`fa-plus` :guilabel:`(+)` icon to the right of a collapsed lot number line to
+   :ref:`manually modify the cost <inventory/warehouses_storage/update-unit-price>`.
+
+   This is useful for adjusting individual lot prices when a purchase order or bill includes
+   multiple lots/serial numbers, as initial prices are identical upon reception.
+
+.. image:: valuation_by_lots/stock-valuation.png
+   :align: center
+   :alt: Show valuation report, by lots.
+
+Valuation smart button
+----------------------
+
+To access a filtered part of the *Stock Valuation* report specific to a lot or serial number, go to
+:menuselection:`Inventory app --> Products --> Lots/Serial Numbers` and select the desired item.
+
+On a lot or serial number page and click the :guilabel:`Valuation` smart button.
+
```

Comment:
I can't see this button on anything in Runbot, so perhaps you can explain what they will see and show an example of what is shown when you click the button?

Reply:
Added an image! :D